website/docs/r/extension_servicenow.html.markdown

```diff
@@ -0,0 +1,87 @@
+---
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_extension_servicenow"
+sidebar_current: "docs-pagerduty-resource-extension-servicenow"
+description: |-
+  Creates and manages a ServiceNow service extension in PagerDuty.
+---
+
+# pagerduty\_extension\_servicenow
+
+A special case for [extension](https://v2.developer.pagerduty.com/v2/page/api-reference#!/Extensions/post_extensions) for ServiceNow.
+
+## Example Usage
+
+```hcl
+data "pagerduty_extension_schema" "webhook" {
+  name = "Generic V2 Webhook"
+}
+
+resource "pagerduty_user" "example" {
+  name  = "Howard James"
+  email = "howard.james@example.domain"
+  teams = [pagerduty_team.example.id]
```

Comment:
The `teams` field is actually deprecated in the `pagerduty_user` resource. For these docs you can just remove the field.